bluetooth/sql/functions/reader_status_history.sql

```diff
@@ -0,0 +1,118 @@
+-- FUNCTION: bluetooth.reader_status_history(date)
+
+-- DROP FUNCTION bluetooth.reader_status_history(date);
+
+CREATE OR REPLACE FUNCTION bluetooth.reader_status_history(
+	insert_value date)
+    RETURNS void
+    LANGUAGE 'plpgsql'
+
+    COST 100
+    VOLATILE 
+AS $BODY$	
+	begin
+		--return query
+			with x AS 
+			(select DISTINCT(analysis_id), 
+			 CASE
+			 WHEN MAX (datetime_bin::date)>= (insert_value-1) then (insert_value-1)
```

Comment:
This comparison is wrong. `insert_value` is the `execution_date`, which would be the day over which the datetime_bins came in. So there shouldn't be `-1` in this comparison.